src/gl_context.rs

```diff
@@ -224,13 +237,17 @@ impl<Native> GLContext<Native>
         self.extensions.clone()
     }
 
+    fn reset_draw_buffer_contents(&self) {
+        self.gl().clear_color(0.0, 0.0, 0.0, if !self.attributes.alpha { 1.0 } else { 0.0 });
```

Comment:
This looks like a separate fix / change, is it needed? If so, a comment in the commit message or the code would be appreciated.